english/sleep/crackers/cracker-soldiers-flee/cracker-soldiers-flee.md

```diff
@@ -0,0 +1,5 @@
+You think to yourself, "Self, this isn't going well."
+
+You eye the biggest, baddest soldier in the room, a gorilla looming over you, and bravely turn tail and flee.
+
+As you push past a rather unintimidating Platypus, something grabs your arm firmly and pulls you back into the circle... 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.